### PR TITLE
Improve/other scenes update

### PR DESCRIPTION
**Pull request by pleszkiewicz**
Update of some more scenes.

### CODE REVIEW

File: src/test_scenes/sceneButtonType.js [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
-				<Text fontSize={0.04}> Icon button</Text>	
-				<Button type='icon' textSide='left' iconType='clock' height={0.15}>text</Button>
-				<Text fontSize={0.04}> Text button</Text>	
-				<Button type='text' textSide='left' iconType='clock' height={0.15}>text</Button>
-			</LinearLayout>
-		</View>
+      <View position={this.props.position}>
+        <LinearLayout 
+        anchorPoint={AnchorPoint.topCenter} 
+        defaultItemAlignment={Alignment.centeerLeft}
```
Comment (kpiascik) [inline src/test_scenes/sceneButtonType.js]: Is his spelled correctly?

File: src/test_scenes/sceneLine.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { Button, View, Line, Text } from 'magic-script-components';
+import { Button, Line, LinearLayout, Orientation, Text, View } from 'magic-script-components';
```
Comment (kpiascik) [inline src/test_scenes/sceneLine.js]: we don't need `View` anymore

File: src/test_scenes/scenePageView.js [hunk 3]
```diff
@@ -59,8 +59,8 @@ class ScenePageView extends React.Component {
     ];
     return (
       <View position={this.props.position}>
-        <LinearLayout defaultItemAlignment={'top-center'} anchorPoint={'top-center'} orientation={'vertical'}>
-          <LinearLayout defaultItemPadding={[0.0, 0.0, 0.0, 0.0]} orientation={'horizontal'}>
+        <LinearLayout defaultItemAlignment={Alignment.topCenter} anchorPoint={AnchorPoint.topCenter} orientation={Orientation.vertical}>
```
Comment (kpiascik) [inline src/test_scenes/scenePageView.js]: we don't need `orientation` here since vertical is the default.

File: src/test_scenes/sceneWebView.js [hunk 4]
```diff
@@ -39,29 +39,30 @@ class SceneWebView extends React.Component {
...
-            text={text}
-            fontColor={[1,1,1,0.75]}
-            fontSize={0.05}
-            width={web.width}
-          />
-          <Button height={0.1} onClick={this.onLoad} fontSize={0.08} >Load</Button>
-          <WebView url={url} height={web.height} width={web.width} />
-          <Button height={0.1} onClick={this.onOpenUrl} fontSize={0.08} >Open in browser</Button> 
-        </LinearLayout>
+        debug
```
Comment (kpiascik) [inline src/test_scenes/sceneWebView.js]: debug?

File: src/test_scenes/sceneButtonType.js (hunk 1 above)
Comment (pleszkiewicz) [reply src/test_scenes/sceneButtonType.js]: No, there is a typo which shows me that PropTypes are not defined for LinearLayout (and some more components).
Good catch 👍

File: src/test_scenes/scenePageView.js (hunk 3 above)
Comment (pleszkiewicz) [reply src/test_scenes/scenePageView.js]: updated

File: src/test_scenes/sceneWebView.js (hunk 4 above)
Comment (pleszkiewicz) [reply src/test_scenes/sceneWebView.js]: This prop turns debug drawings on. I can turn it off but in many other scenes I keep this flag on to visualize how layouts work.

**Review by kpiascik — APPROVED** [review]

**Pull request merged**